core/src/main/scala/org/apache/spark/package.scala

```diff
@@ -43,5 +43,5 @@ package org.apache
 
 package object spark {
   // For package docs only
```

Comment:
BTW, not your fault, and I'm not sure what exactly this comment refers to, but it's misleading. `SPARK_VERSION` is definitely not used only for package docs.